packages/errors/src/test-utils/expect-to-throw-fuel-error.test.ts

```diff
@@ -0,0 +1,12 @@
+import { FuelError } from '..';
+
+import { expectToThrowFuelError } from './expect-to-throw-fuel-error';
+
+
+describe('FuelError', () => {
+  it('should compare whole or partial error objects', () => {
+    const error = new FuelError(FuelError.CODES.PARSE_FAILED, 'It happens')
+    expectToThrowFuelError(() => {throw error}, error)
+    expectToThrowFuelError(() => {throw error}, { code: FuelError.CODES.PARSE_FAILED })
+  });
```

Comment:
```suggestion
  });
  
  it('should fail with non-fuel errors', () => {
    const expected = { code: FuelError.CODES.PARSE_FAILED };
    const fnThrower = () => {
      throw new Error('x'); // not a FuelError
    };
    expect(expectToThrowFuelError(fnThrower, expected)).rejects.toThrow(
      /Expected constructor:.+FuelError/
    );
  });

  it('should fail on the absence of errors', () => {
    const code = FuelError.CODES.PARSE_FAILED;
    const fnThrower = () => {
      /* no throws here */
    };
    expect(expectToThrowFuelError(fnThrower, { code })).rejects.toThrow(
      /Received value:.+undefined/
    );
  });
```

Reply:
I've taken the 'should fail with non-fuel errors' test but not the second one as it was already handled in 'fails when passed fn doesnt throw'